src/reports/components/report-sections/no-failed-instances-congrats.tsx

```diff
@@ -5,16 +5,30 @@ import * as React from 'react';
 import { InlineImage, InlineImageType } from 'reports/components/inline-image';
 import * as styles from './no-failed-instances-congrats.scss';
 
-export const NoFailedInstancesCongrats = NamedFC('NoFailedInstancesCongrats', () => {
-    return (
-        <div className={styles.reportCongratsMessage}>
-            <div className={styles.reportCongratsHead}>Congratulations!</div>
-            <div className={styles.reportCongratsInfo}>No failed automated checks were found.</div>
-            <InlineImage
-                className={styles.sleepingAda}
-                imageType={InlineImageType.SleepingAda}
-                alt=""
-            />
-        </div>
-    );
-});
+export type NoFailedInstancesCongratsDeps = {
+    customCongratsMessage: String;
+};
+
+export type NoFailedInstancesCongratsProps = {
+    deps: NoFailedInstancesCongratsDeps;
+};
+
+export const NoFailedInstancesCongrats = NamedFC<NoFailedInstancesCongratsProps>(
+    'NoFailedInstancesCongrats',
+    props => {
+        const message = props.deps.customCongratsMessage
```

Comment:
you might be able to use ?? here instead of ternary

Reply:
Will do this when I fix the merge conflict